src/index.css

```
@@ -373,75 +651,259 @@
 
   .wiki-link:hover {
     background: hsl(var(--link-color) / 0.2);
   }
 
-  /* Referenced link - exists in other pages but no page created yet */
+  /* Referenced — exists in other pages but not yet created */
   .wiki-link-referenced {
-    @apply text-link-referenced -mx-0.5 cursor-pointer rounded border-b border-dashed border-current px-0.5 font-medium transition-colors duration-200 hover:border-solid;
+    color: hsl(var(--link-referenced));
     background: hsl(var(--link-referenced) / 0.1);
+    border-bottom: 1px dashed currentColor;
```

Stylelint (17.9.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)